src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -394,7 +410,8 @@ std::vector<std::optional<PosHashTable<HashedType>>> build(const RadixContainer<
     for (size_t partition_idx = 0; partition_idx < radix_container.size(); ++partition_idx) {
       total_size += radix_container[partition_idx].elements.size();
     }
-    hash_tables = {PosHashTable<HashedType>(mode, total_size)};
+    hash_tables.resize(1);
+    hash_tables[0] = PosHashTable<HashedType>(mode, total_size);
```

Comment:
No. I couldn't get it to compile otherwise. Something with the move constructors. I don't think anyone will notice this when not looking at this diff, so a comment would likely confuse more than it would help.

Reply:
As long as there is no specific reason for writing it this way, that is fine for me.